.eslintrc.json

```diff
@@ -25,15 +25,15 @@
   "plugins": ["react", "react-native", "jest"],
   "ignorePatterns": ["**/__mocks__/*"],
   "rules": {
-    // General ESLINT rules
-    "indent": ["error", 2, { "SwitchCase": 1 }],
-    "linebreak-style": ["error", "unix"],
-    "quotes": ["error", "single"],
-    "semi": ["error", "always"],
-    "no-unused-vars": ["warn", { "varsIgnorePattern": "(React)" }],
-    "eol-last": ["error", "always"],
-    "no-multiple-empty-lines": ["error", { "max": 1 }],
-    "brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
+      // General ESLINT rules
```

Comment:
somehow these rules are indented more than the following lines?

Reply:
fixed in later commit as rebasing issues